wormhole-connect/src/views/Bridge/TransferLimitedWarning.tsx

```diff
@@ -1,25 +1,219 @@
+    paddingTop: 24,
+    paddingBottom: 24,
+  },
+  title: {
+    textAlign: 'center',
+    fontSize: 24,
+    fontWeight: 600,
+    lineHeight: '32px',
+    paddingTop: 24,
+    paddingBottom: 24,
+    borderBottom: '1px solid #313266',
+  },
+  text: {
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: '24px',
+    color: '#9495D6',
+    paddingTop: 24,
+    paddingBottom: 24,
+  },
+  link: {
+    color: '#D5D5EB',
+    textDecorationLine: 'underline',
+  },
+  orange: {
+    color: '#E48329',
+  },
+  clock: {
+    width: 64,
+    height: 64,
+  },
+  buttonContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginTop: '24px',
+    gap: 10,
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+}));
+
+const learnMoreLink =
+  'https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0007_governor.md';
 
 const TransferLimitedWarning = () => {
+  const { classes } = useStyles();
+  const [open, setOpen] = React.useState(true);
   const isTransferLimited = useIsTransferLimited();
+
+  const onClose = () => {
+    setOpen(false);
+  };
+
+  const onContinue = () => {
+    setOpen(false);
+    setTimeout(() => {
+      window.scrollTo(0, document.body.scrollHeight);
+    }, 500);
+  };
+
   if (
     isTransferLimited.isLimited &&
     isTransferLimited.reason &&
     isTransferLimited.limits
   ) {
     const chainName = wh.toChainName(isTransferLimited.limits.chainId);
-    const message =
-      isTransferLimited.reason === 'EXCEEDS_MAX_NOTIONAL'
-        ? `This transfer's estimated notional value would exceed the notional value limit for transfers on ${chainName} (${isTransferLimited.limits.chainNotionalLimit}) and may be subject to a 24 hour delay.`
-        : isTransferLimited.reason === 'EXCEEDS_LARGE_TRANSFER_LIMIT'
-        ? `This transfer's estimated notional value may exceed the notional value for large transfers on ${chainName} (${isTransferLimited.limits.chainBigTransactionSize}) and may be subject to a 24 hour delay.`
-        : isTransferLimited.reason === 'EXCEEDS_REMAINING_NOTIONAL'
-        ? `This transfer's estimated notional value may exceed the remaining notional value available for transfers on ${chainName} (${isTransferLimited.limits.chainRemainingAvailableNotional}) and may be subject to a delay.`
-        : '';
-    return <AlertBanner show={!!message} content={message} error />;
+
+    let message, title;
+    if (
+      isTransferLimited.reason === 'EXCEEDS_MAX_NOTIONAL' ||
+      isTransferLimited.reason === 'EXCEEDS_REMAINING_NOTIONAL'
+    ) {
+      // TODO: See if its necessary a different message for EXCEEDS_REMAINING_NOTIONAL case
+      title = (
+        <>
+          This transaction will take up to{' '}
+          <span className={classes.orange}>24 hours</span> to complete
+        </>
+      );
+      message = (
+        <>
+          This transaction will take up to 24 hours to complete. This{' '}
```

Comment:
Can consolidate these two sentences and say, This transaction will take up to 24 hours to complete as Wormhole has reached the daily limit for {chainName}.

Reply:
@mliu-xlabs What do you think about this option?

Reply:
ok, the first sentence corresponded to the title, so now there is no repetition @kev1n-peters